### DISCUSSION TITLE
Is there a feature that makes it impossible for other players to play even if they know all the token values?

### DISCUSSION THREAD

**Discussion by Jamong435** [ant-media/Ant-Media-Server#6024] [FAQs]
Hello, I've explored using tokens for secure streaming with ant media. Is there a way to prevent playback even if another user manages to acquire all the token values? Is this functionality only achievable with one-time tokens?

**Comment by USAMAWIZARD** [comment]
Hi @Jamong435 , 
You can use Subscriber Block for this [here](https://github.com/orgs/ant-media/discussions/5796#discussioncomment-7577999) is the detailed guide on subscriber block.
